install/Knative-with-AKS.md

```diff
@@ -142,46 +142,24 @@ rerun the command to see the current status.
 > Note: Instead of rerunning the command, you can add `--watch` to the above
   command to view the component's status updates in real time. Use CTRL + C to exit watch mode.
 
-## Installing Knative components
+## Installing Knative
 
-You can install the Knative Serving and Build components together, or Build on its own.
-
-### Installing Knative Serving and Build components
+The following commands install all available Knative components. To customize
+your Knative installation, see [Performing a Custom Knative Installation](Knative-custom-install.md).
 
 1. Run the `kubectl apply` command to install Knative and its dependencies:
     ```bash
-    kubectl apply --filename https://github.com/knative/serving/releases/download/v0.2.1/release.yaml
+    kubectl apply --filename https://github.com/knative/serving/releases/download/v0.2.1/release.yaml /
+    --filename https://github.com/knative/eventing/releases/download/v0.2.0/eventing.yaml /
     ```
 1. Monitor the Knative components until all of the components show a
    `STATUS` of `Running`:
     ```bash
     kubectl get pods --namespace knative-serving
```

Comment:
The https://github.com/knative/serving/releases/download/v0.2.2/release.yaml also include a namespace named as `knative-monitoring`, I think we should also list it here. Ditto for other install guides.

```
kubectl get pods --namespace knative-monitoring
```

Reply:
I think here we really only want to take a look at the pods for the main three components. I'm not 100% sold on including knative-sources...

@RichieEscarez  what are your thoughts on what pods we monitor at the end of the install guides?

Reply:
with the exception of release-no-mon.yaml (for all release*.yaml manifests) the knative-monitoring namespace gets created and it should get looked at to ensure it installed okay (so add `kubectl get pods --namespace knative-sources` to the list)